client/landing/jetpack-cloud/components/date-picker/index.jsx

```diff
@@ -52,27 +48,39 @@ class DatePicker extends Component {
 		return moment( date ).format( dateFormat );
 	};
 
-	shuttleLeft = () => {
+	goToPreviousDay = () => {
+		if ( ! this.canGoToPreviousDay() ) {
+			return false;
+		}
 		const { moment, onDateChange, selectedDate } = this.props;
 
 		const newSelectedDate = moment( selectedDate ).subtract( 1, 'days' );
 
 		onDateChange( newSelectedDate.toDate() );
 	};
 
-	shuttleRight = () => {
-		if ( ! this.canShuttleRight() ) {
+	goToNextDay = () => {
+		if ( ! this.canGoToNextDay() ) {
 			return false;
 		}
-
 		const { moment, onDateChange, selectedDate } = this.props;
 
 		const newSelectedDate = moment( selectedDate ).add( 1, 'days' );
 
 		onDateChange( newSelectedDate.toDate() );
 	};
 
-	canShuttleRight = () => {
+	canGoToPreviousDay = () => {
+		const { moment, selectedDate, oldestDateAvailable } = this.props;
+
+		if ( false === !! oldestDateAvailable ) {
+			return true;
+		}
+
+		return ! moment( selectedDate ).isSame( oldestDateAvailable, 'day' );
```

Comment:
I feel like we could collapse this function into a one-liner with a ternary:
```suggestion
		return ! oldestDateAvailable ? true :  ! moment( selectedDate ).isSame( oldestDateAvailable, 'day' );
```

Reply:
And even better: `return ! oldestDateAvailable || ! moment( selectedDate ).isSame( oldestDateAvailable, 'day' );`